corehq/apps/cloudcare/static/cloudcare/js/utils.js

```diff
@@ -129,7 +129,11 @@ hqDefine('cloudcare/js/utils', [
     };
 
     var formplayerLoading = function () {
-        showLoading();
+        var validationInProgress= sessionStorage.validationInProgress === undefined ?
```

Comment:
```suggestion
        var validationInProgress = sessionStorage.validationInProgress === undefined ?
```
otherwise this looks good.